src/handlers/payout/post.ts

```diff
@@ -121,6 +131,138 @@ export const incentivizeComments = async () => {
   await addCommentToIssue(comment, issue.number);
 };
 
+export const incentivizePullRequestReviews = async () => {
+  const logger = getLogger();
+  const {
+    mode: { incentiveMode, paymentPermitMaxPrice },
+    price: { baseMultiplier, incentives },
+    payout: { paymentToken, rpc },
+  } = getBotConfig();
+  if (!incentiveMode) {
+    logger.info(`No incentive mode. skipping to process`);
+    // return;
+  }
+  const context = getBotContext();
+  const payload = context.payload as Payload;
+  const issue = payload.issue;
+  if (!issue) {
+    logger.info(`Incomplete payload. issue: ${issue}`);
+    return;
+  }
+
+  if (issue.state_reason !== StateReason.COMPLETED) {
+    logger.info("incentivizeComments: comment incentives skipped because the issue was not closed as completed");
+    // return;
+  }
+
+  if (paymentPermitMaxPrice == 0 || !paymentPermitMaxPrice) {
+    logger.info(`incentivizeComments: skipping to generate permit2 url, reason: { paymentPermitMaxPrice: ${paymentPermitMaxPrice}}`);
+    // return;
+  }
+
+  const issueDetailed = bountyInfo(issue);
+  if (!issueDetailed.isBounty) {
+    logger.info(`incentivizeComments: its not a bounty`);
```

Comment:
Doesn't it make more sense to refactor this `logger` method to include a stack trace? We should also not be using anonymous functions (instead do `async function incentivizePullRequestReviews() { ...`) and the function name, file number, and line number will appear in the logs.

Reply:
Do you mean, replacing logger to console.trace?

And also not sure about `using anonymous functions` 
would you mind giving some tips about it?

Reply:
instead do a traditional function declaration

```typescript
async function incentivizePullRequestReviews() { } 
```

then the function has a name which will appear clearly in the stack trace.

you can probably extract the stack trace from the error object or from console.trace and dump the contents into the logger. then you dont need to prefix every log manually like you are doing here with "incentivizeComments: "

Reply:
hmm done.